src/test/java/seedu/moneygowhere/model/spending/DateTest.java

```diff
@@ -34,4 +35,11 @@ public void isValidDate() {
         assertTrue(Date.isValidDate("today"));
         assertTrue(Date.isValidDate("yesterday"));
     }
+
+    @Test
+    public void compareTo() {
```

Comment:
Would be good if cana add test for dates across different years e.g 31/12/2018 and 1/1/2019

Reply:
Thanks, will add it into the test cases!